<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add PlatList</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="UrlLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Url&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="PlayListLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;PlayList&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="InputUrl">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="InputPlayList">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>118</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
